### PR TITLE
Multi-function return

### PR DESCRIPTION
**Pull request by scottdtaylor95**

### CODE REVIEW

File: syserol/regression.py [hunk 1]
```diff
@@ -11,53 +11,58 @@
...
 
 
-def function_prediction(tensorFac, function="ADCC", evaluation="all"):
+def function_prediction(tensorFac, evaluation="all"):
     """ Predict functions using our decomposition and regression methods"""
     func, _ = importFunction()
     indices = AlterIndices()
+    our_function_results = []
+    for function in functions:
+        Y = func[function]
```
Comment (aarmey) [inline syserol/regression.py:43]: If you put all the functions as columns in Y, you can predict them all at once (without a loop).

**Review by aarmey — CHANGES_REQUESTED** [review]

File: syserol/regression.py (hunk 1 above)
Comment (scottdtaylor95) [reply syserol/regression.py:43]: Maybe I'm confused, but wouldn't I still need a loop for the rest of the function?

Comment (aarmey) [reply syserol/regression.py:43]: No, you can actually do everything here without loops, and that would be ideal for adjusting this later. For example, look at [`r2_score`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.r2_score.html). If you give it `Y`'s that are matrices, it understands that the rows are samples and the columns are different outputs, then calculates the values for each output.

Comment (scottdtaylor95) [reply syserol/regression.py:43]: Okay, how do I deal with the fact that for some functions, certain subjects are dropped due to NaN while others aren't?

Comment (aarmey) [reply syserol/regression.py:43]: Ergh. Then we need the loop. Will review again in a bit.

**Review by aarmey — APPROVED** [review]
Good to merge.

**Pull request closed without merging**